ChameleonForms/FieldGenerators/Handlers/DateTimeHandler.cs

```diff
@@ -36,7 +37,15 @@ public override IHtmlString GenerateFieldHtml(IReadonlyFieldConfiguration fieldC
         public override void PrepareFieldConfiguration(IFieldConfiguration fieldConfiguration)
         {
             if (!string.IsNullOrEmpty(FieldGenerator.Metadata.DisplayFormatString))
-                fieldConfiguration.Attr("data-val-format", FieldGenerator.Metadata.DisplayFormatString.Replace("{0:", "").Replace("}", ""));
+            {

```

Comment:
We have 
```
[DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = @"{0:g}")]
````
in our ViewModel. And I want validation of the field according UICulture of web-server. To accomplish that and make the jquery.validate.unobtrusive.chameleon.js to work, I make this injection to data-val-format.